features/guidance/turn-lanes.feature

```diff
@@ -107,6 +107,24 @@ Feature: Turn Lane Guidance
             | a,d       | road,turn,turn | depart,turn right,arrive        | ,straight:false right:true, |
             | a,c       | road,road,road | depart,use lane straight,arrive | ,straight:true right:false, |
 
+    @PROFILE @LANES
```

Comment:
On my system, this fails to trigger any issues. I've added the test nonetheless.